src/pages/about.astro

```diff
@@ -4,300 +4,219 @@ import CtaButtom from "../components/InlineCtaButton.astro"
-        <CtaButtom className="about-button" link="/contact">Get in Touch</CtaButtom>
-    </p>
-</div>
+	<p class='big-text'>
+		At Vivic Research, we support agents working toward social justice with economic
+		and data-driven research. We serve advocates, think tanks, governments, labour
+		unions, and other agents engaged in the development of public policy. We empower
+		our clients with economic knowledge through our practice areas: economic research,
+		data science, and training.
+	</p>
+	<div id="about-section">
+		<section>
+			<h3>
+				Our Vision
+			</h3>
+			<p>
+				Vivic Research envisions a just society, where everyone has access
+				to housing, food, safety, health and community.
+			</p>
+		</section>
+		<section>
+			<h3> 
+				Our Mission 
+			</h3>
+			<p> 
+				Vivic Research’s mission is provide alternatives to traditional economic
+				consulting. We work towards our collective liberation by actively
+				challenging standard assumptions and existing systems.
+			</p>
+		</section>
+	</div>
+
+	<h2>Our Team</h2>
+	<div class="team">
+		<section>
+			<picture>
+				<source type="image/avif" srcset="/images/inez.avif"/>
+				<source type="image/webp" srcset="/images/inez.webp"/>
+				<img src="/images/inez.jpg" alt="inez">
+			</picture>
+			<h3>
+				Inez Hillel <span class="pronouns">(she/her)</span>
+			</h3>
+			<h4>
+				Economist and Co-founder
+			</h4>
+			<p>
+				Inez Hillel is co-founder and economist of Vivic Research. She currently
+				works on the traditional territory of the Huron-Wendat, the Seneca
+				and the Mississaugas of the Credit. Inez is passionate about intersectional
+				economics that center frontline voices and qualitative data as foundational
+				for progressive policy development. Her focus at Vivic is projects
+				that require ongoing community engagement and that allow her to connect
+				with grassroots activists and organizations. Having previously worked
+				at the Centre for the Study of Living Standards, she has conducted
+				research on income supports in Canada.  Inez holds an MA in Economics
+				from Simon Fraser University and a BA in Applied Economics from Queen’s
+				University. In 2020, Inez was the recipient of the British Columbia
+				Graduate Scholarship and the Provost Prize of Distinction Scholarship
+				from Simon Fraser University. 
+			</p>
+			<CtaButtom className="about-button" link="/contact">
+				Get in Touch
+			</CtaButtom>
```

Comment:
I moved the contact link out of the `<p>`. I don't think that's strictly more or less semantic, but it looks better in code.